Makefile

```diff
@@ -84,6 +88,7 @@ DIR_SOURCES_LEXER_QUOTE = sources/lexer/quote
 DIR_SOURCES_PARSER		= sources/parser
 DIR_SOURCES_UTILS		= sources/utils
 DIR_SOURCES_EXECUTOR	= sources/executor
+DIR_SOURCES_EXECUTOR_BUILTINS = sources/executor/builtins
```

Comment:
Perhaps the builtins should be in their own directory. The executor is responsible for the execution of functions. The built ins are functions themselves, and should have their own root directory.